src/main/java/core/selenium/utils/IdGenerator.java

```diff
@@ -0,0 +1,26 @@
+package core.selenium.utils;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class IdGenerator {
+
+    static DateFormat hour;
```

Comment:
probably this fields will be used only in getGeneratedId, they should be inside the method